src/components/Marketplace/Marketplace.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {Navigation} from "../index";
+import PTProfile from "../Profile/PTProfile";
+
+function Marketplace() {
```

Comment:
I think that this is the old marketplace file and should be deleted. We are using Marketplace.js and not Marketplace.jsx now.

Reply:
ah yes I accidentally forgot to delete it, it's not being used though

Reply:
oh ok. just remember to delete the file from the branch before you merge it to main.

Reply:
deleted! :)